src/Sylius/Bundle/FixturesBundle/DataFixtures/DataFixture.php

```diff
@@ -118,4 +130,31 @@ protected function get($id)
     {
         return $this->container->get($id);
     }
+
+    /**
+     * @return AddressInterface
+     */
+    protected function createAddress()
```

Comment:
We will need fake address in several fixtures so that can be useful. (I used it in stock location etc.)